pkgs/poetry/update

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env nix-shell
 #! nix-shell -i bash -p curl nix-prefetch-github jq
 
-rev=$(curl -s https://api.github.com/repos/python-poetry/poetry/releases/latest | jq -r '.name')
+rev=1.2.0a2
```

Comment:
Doesn't this defeat the purpose of this script?

Reply:
I shouldn't have added that to the commit, it was a oneshot thing.